lib/connectors/memory.js

```diff
@@ -679,6 +681,52 @@ Memory.prototype.updateAttributes = function updateAttributes(model, id, data, o
   }
 };
 
+Memory.prototype.replace = function replace(model, id, data, options, cb) {
+  var self = this;
+  if (!id) {
+    var err = new Error('You must provide an id when replacing!');
+    if (cb) {
+      return cb(err);
+    } else {
+      throw err;
+    }
+  }
+
+  // Do not modify the data object passed in arguments
+  // data = Object.create(data);
```

Comment:
After this line of code `data = Object.create(data);` data will be an empty object; not sure why!